pdr_backend/statutil/assets/styles.css

```diff
@@ -5,3 +5,7 @@
     background-color: #2b2a26;
     text-align: start;
 }
+
+[class*="table-container"]{
```

Comment:
Here too, I suggest using the container and column/row structures from dbc. Easier to maintain and pretty much standard since it's Bootstrap underneath. Also gives an uniform feel to our dashboards.

Reply:
I don't really like using Bootstrap components because they don't provide the flexibility and control that I want. They are faster at higher lvl styling but when you want to do something more custom you end up overriding a lot.